src/styles/typography.js

```diff
@@ -0,0 +1,63 @@
+import { createGlobalStyle } from "styled-components"
+
+export const primaryFont = "GT America"
+
+const Typography = createGlobalStyle`
+    @font-face {
+        font-family: ${primaryFont};
+        font-weight: 400;
+        src: url("/static/GT-America-Standard-Regular.woff2") format("woff2"),
+            url("/static/GT-America-Standard-Regular.ttf") format("truetype"),
+            url("/static/GT-America-Standard-Regular.otf") format("opentype");
+    }
+
+    @font-face {
+        font-family: ${primaryFont};
+        font-weight: 700;
+        src: url("/static/GT-America-Standard-Bold.woff2") format("woff2"),
+            url("/static/GT-America-Standard-Bold.ttf") format("truetype"),
+            url("/static/GT-America-Standard-Bold.otf") format("opentype");
+    }  
+    
+    * {
+        font-family: system-ui, "Helvetica Neue", sans-serif;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+    font-weight: 400;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    }
+
+    /* Type Scale */
```

Comment:
All of this should really just be in the theme file now that it's implemented, right? We should also be adding spacing values into the theme file, too to improve consistency.

This is something I can tackle after merge 😄

Reply:
It's totally valid to put your font-sizes and weights in the theme.js file. The typography.js file simply servers as an assurance that certain root styles are applied globally. That way you don't have too redefine header elements or the root font-size in every component as needed.